08-VendoringDependenciesAsSubmodules.adoc

```diff
@@ -3,7 +3,7 @@ ifdef::env-github[:outfilesuffix: .adoc]
 
 In this chapter you will learn how to maintain dependencies on other Git-based software projects within your own using Git submodules.
 
-* When submodules are useful
+* When are submodules useful
```

Comment:
Please leave this as-is, thanks!

```suggestion
* When submodules are useful
```

Reply:
OK. I put in in question form like the following bullets, but I'll revert that.